src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,855 @@
+        /// The external reference voltage value. The default is 2.5V on REFIN1+ and REFIN1- (on the Digilent Pmod AD5 board)
+        /// </summary>
+        public double VReference { get; set; } = 2.50f;
+
+        /// <summary>
+        /// Polarity select bit: true is unipolar, false is bipolar operation.
+        /// </summary>
+        public bool Unipolar
+        {
+            get
+            {
+                return (GetRegisterBits(Register.Configuration, BitMask.ConfigurationU) == 1);
+            }
+
+            set
+            {
+                _registerCache[(byte)Register.Configuration] = SetRegisterBits(Register.Configuration, BitMask.ConfigurationU, value ? 1U : 0);
+                SetRegisterValue(Register.Configuration, _registerCache[(byte)Register.Configuration]);
+            }
+        }
+
+        /// <summary>
+        /// Gain level on the ADC
+        /// </summary>
+        public Gain PGAGain
+        {
+            get
+            {
+                return (Gain)(GetRegisterBits(Register.Configuration, BitMask.ConfigurationG));
+            }
+
+            set
+            {
+                _registerCache[(byte)Register.Configuration] = SetRegisterBits(Register.Configuration, BitMask.ConfigurationG, (uint)value);
+                SetRegisterValue(Register.Configuration, _registerCache[(byte)Register.Configuration]);
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Status register
+        /// </summary>
+        public uint Status
+        {
+            get
+            {
+                _registerCache[(byte)Register.Status] = GetRegisterValue(Register.Status);
+                return _registerCache[(byte)Register.Status];
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Mode register
+        /// </summary>
+        public uint Mode
+        {
+            get
+            {
+                _registerCache[(byte)Register.Mode] = GetRegisterValue(Register.Mode);
+                return _registerCache[(byte)Register.Mode];
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Configuration register
+        /// </summary>
+        public uint Config
+        {
+            get
+            {
+                _registerCache[(byte)Register.Configuration] = GetRegisterValue(Register.Configuration);
+                return _registerCache[(byte)Register.Configuration];
+            }
+        }
+
+        /// <summary>
+        /// Power-down mode. In power-down mode, all AD7193 circuitry, except the bridge power-down switch, is powered down. The bridge power-down switch remains active because the user may need to power up the sensor prior to powering up the AD7193 for settling reasons. The external crystal, if selected, remains active.
+        /// </summary>
+        public bool IsPowerDown
+        {
+            get
+            {
+                GetRegisterValue(Register.Mode);
```

Comment:
is the return value ignored intentionally?

Reply:
yes